components/admin_console/billing/billing_history.tsx

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import FormattedAdminHeader from 'components/widgets/admin_console/formatted_admin_header';
+
+type Props = {
+
+};
+
+const BillingHistory: React.FC<Props> = () => {
+    return (
+        <div className='wrapper--fixed BillingHistory'>
+            <FormattedAdminHeader
+                id='admin.billing.history.title'
+                defaultMessage='Billing History'
+            />
+            <div className='admin-console__wrapper'>
+                <div className='admin-console__content'>
+                    <div style={{border: '1px solid #000', width: '100%', height: '463px'}}>
```

Comment:
should the styling be moved to its own scss file?

Reply:
All the in-line styles are temporary to show where everything goes.

Reply:
Ahh gotcha.